test/elm/list/list-test.ts

```diff
@@ -671,37 +671,37 @@ describe('First', () => {
 
   it('should get first of a list of numbers and the localId should exist on the root context', async function () {
     (await this.numbers.exec(this.ctx)).should.equal(1);
-    should(this.ctx.localId_context[6]).not.be.undefined();
+    should(this.ctx.localId_context[213]).not.be.undefined();
```

Comment:
Oh man. Fixing these must have been pretty annoying.

Also... technically speaking, `localId` is a _string_ -- and when we use it to add a key to the `localId_context`, we keep it a string (see [setLocalIdWithResult](https://github.com/cqframework/cql-execution/blob/implies-operator/src/runtime/context.ts#L174-L180)). So we probably _should_ access it using a string key as well (e.g., `this.ctx.localId_context['213']`). I guess it works if you pass in a number anyway (because javascript), but it leads to confusion since it makes it look like an array accessor. I know this is totally _not your fault_ (it was already like this), but... what do you think of fixing it?

Or... do you think we should consider creating a helper function to grab the right local id from the data file? It looks like we could define a function like this:
```ts
function getListLocalId(testName: string, statementName: string) {
  return data[testName].library.statements.def.find((d: any) => d.name === statementName).expression
    .source.localId;
}
```
and then in the tests do something like this (which would work in this test that I'm commenting on):
```ts
const localId = getListLocalId('First', 'Numbers');
should(this.ctx.localId_context[localId]).not.be.undefined();
```
This would mean that if a translator change (or a change in the test data CQL file) results in changing all the localIds, we wouldn't have to do this tedious task of fixing all the numbers again. On the other hand, it's not as explicit as just putting in the literal localId. What do you think?

Reply:
I ended up getting a bit more generic with this since it wasn't always the first List item that was being looked for. I made a helper function that would traverse the ELM given a path to traverse and return the localId if it was successful.

Reply:
Ah, good call!